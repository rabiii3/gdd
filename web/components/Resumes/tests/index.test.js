import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from '../../../store/configureStore.test';
import { enhance } from '..';

describe('app | components | Resumes | List | index', () => {
  describe('render', () => {
    it('should render', () => {
      const Component = () => <div />;
      const EnhancedComponent = enhance(Component);
      const state = {
        people: {
          data: {
            '1': {
              _id: '1',
              firstname: 'james',
              lastname: 'bond',
              email: 'james.bond@gmail.com',
              status: 'desable',
              type: 'user',
              phoneNumber: '07737383939',
              color: 'red',
              roles: ['user'],
            },
          },
        },
        resumes: {
          data: [
            {
              _id: 2,
              status: 'pending',
              //updatedAt: new Date(),
              candidateId: 1,
              createdBy: '4',
              //createdAt: new Date(),
            },
          ],
        },
      };
      const store = configureStore(state);

      const wrapper = mount(
        <Provider store={store}>
          <EnhancedComponent />
        </Provider>,
      );

      expect(wrapper).toMatchSnapshot();
    });
  });
});
