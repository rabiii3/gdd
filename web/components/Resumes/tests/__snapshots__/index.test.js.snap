// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | components | Resumes | List | index render should render 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <WithStyles(Component)>
    <Component
      classes={Object {}}
    >
      <div />
    </Component>
  </WithStyles(Component)>
</Provider>
`;
